src/stories/Page.jsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react'
+import './page.css'
 
 import { Header } from './Header'
-import './page.css'
+import React from 'react'
```

Comment:
You'll need to import React at the first line and import css file at the last after the import of the Header.